index.js

```diff
@@ -3,6 +3,8 @@
 var each = require('lodash.foreach');
 var get = require('lodash.get');
 
+var Promise = require('bluebird');
```

Comment:
Thanks for your quick response. Bluebird was already included as a dependency, but I didn't realize it was a devDependency. Bluebird was indeed chosen for performance, but in this case we might be better of using native Promises. I'll push a new version without requiring bluebird!